randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -1249,7 +1275,15 @@ Extra - room_id: [33]
   * L1 Hatch to Pump Control Access/Door to Pump Control Unit
   * Extra - door_idx: (76,)
   > Pickup (Missile Tank)
-      Gravity Suit and Morph Ball and Space Jump and Damage Runs (Beginner)
+      All of the following:
+          Gravity Suit and Morph Ball
+          Any of the following:
+              Space Jump
+              # With HJ + WJ: https://youtu.be/5cASpPGdiig
+              Hi-Jump and Wall Jump (Intermediate)
+          Any of the following:
+              After Sector 4 Water Level Lowered
```

Comment:
i wonder if this should get a comment? you cant currently lower the water level without speed, and if you have speed you are definitely able to just go the other way around.
do feel like its worth keeping just in case:tm: for the future.